bramkouwen/Kouwen_Abram_Mariia_Shevchenko_FIP
# simplifying branch

## What does the PR do?
* It clears un-needed data

## How do you check this PR?
* Pull down the branch
*  See cleared data

## Screenshots 
None.
